doc/semieunit.xml

```diff
@@ -0,0 +1,292 @@
+############################################################################
+##
+#W  semieunit.xml
+#Y  Copyright (C) 2016                                   Christopher Russell
+##
+##  Licensing information can be found in the README file of this package.
+##
+#############################################################################
+##
+
+<#GAPDoc Label="IsMcAlisterTripleSemigroup">
+  <ManSection>
+    <Prop Name = "IsMcAlisterTripleSemigroup" Arg = "S"/>
+    <Returns><K>true</K> or <K>false</K>.</Returns>
+    <Description>
+      A <E>McAlister triple semigroup</E> is an E-unitary inverse semigroup
```

Comment:
I currently have the mathematical details in z-chap15.xml at the start of my section and some of it mentioned again later. Should I rearrange like this:

-Introduction without much detail at start of the section
-Technical details in IsMcAlisterTripleSemigroup
-Mathematical details in McAlisterTripleSemigroup